app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -51,6 +58,92 @@ angular.module('topologyApp', ['kubernetesUI', 'ui.bootstrap'])
     $interval.cancel(promise);
   });
 
+
+  var refreshProvider = function(d) {
+    $scope.refreshProvider.value = true;
+    document.getElementsByClassName('alert alert-success alert-dismissable refresh_provider')[0].style.display ='block';
```

Comment:
@himdel I just uploaded another attempt, it still doesn't work, and I can't figure out why